docs/publishing.md

```diff
@@ -71,7 +71,11 @@ You can install these prereleases using a command such as `npm install web-featu
 > [!NOTE]
 > This information is for [project owners](../GOVERNANCE.md#roles-and-responsibilities).
 
-Publishing requires the `NPM_TOKEN` repository secret (set via _Settings_ → _Secrets and variables_ → _Actions_).
+Publishing requires the `NPM_TOKEN` repository secret.
+
+Set the secret by running `gh secret set --repo=web-platform-dx/web-features NPM_TOKEN`
```

Comment:
```suggestion
Set the secret by running `gh secret set --repo=web-platform-dx/web-features NPM_TOKEN` which will prompt you to paste the secret
```

Optional- my initial thought was that this was missing the actual secret, and went to look at the API docs to figure out the syntax anyways.